### ISSUE TITLE
Provide a pure-floss build flavor

### ISSUE THREAD

**Issue by ghost** [ThibaudM/timelapse-sony#26]
You recently introduced a non-free dependency, play-services. Could you provide a pure floss build flavor that doesn't rely on proprietary software?

**Comment by ThibaudM** [comment]
I only use play services for advertising. 
The main code is not impacted by a non-free dependency.
What do you suggest? Create a new branch without advertising?

**Comment by krt16s** [comment]
As I said, most devs dont want to go through the overhead to manage
two branche, but use gradle build flavors instead. Here is an example
from the docs at https://developer.android.com/studio/build/build-variants.html:

````
android {
    productFlavors {
        demo {
            applicationIdSuffix ".demo"
            versionNameSuffix "-demo"
        }
        full {
            applicationIdSuffix ".full"
            versionNameSuffix "-full"
        }
    }
}
````

You can then specify falvor-specific dependencies, e.g. via
`fullCompile` and `demoCompile`. You also can move code to
only be included in one flavor. By default, a flavor uses everyting
from src/main and src/flavorname.

**Comment by ThibaudM** [comment]
I rolled back the use of play services for advertising.
There are no more dependencies to play services or to another non-free library.

**Comment by relan** [comment]
Also remove `classpath 'com.google.gms:google-services:3.0.0'` from https://github.com/ThibaudM/timelapse-sony/blob/master/build.gradle#L10 please.

**Comment by ThibaudM** [comment]
You are right, this was an old dependency. It has been fixed in b6980e8581ca445b7368f05aed466796a735f87a

**Comment by patzi** [comment]
So this seems to be fixed and can be closed? Maybe you can make a new v3.0.6 tag release and than f-droid will update as well. Thank you for your affords and the app.

**Comment by ThibaudM** [comment]
Done
